build.gradle

```diff
@@ -125,6 +126,9 @@ dependencies {
   // spotbugs
   implementation "com.github.spotbugs:spotbugs-annotations:$spotbugsVersion"
 
+  schemaSpy 'org.schemaspy:schemaspy:6.2.4'
```

Comment:
Does schemaspy licence LGPL-3.0 and GPL-3.0 comply with current EUPL-1.2 license in all possibly generated material?

Reply:
- Overall,  [EUPL-1.2 is compatible with GPL 3.0](https://joinup.ec.europa.eu/collection/eupl/matrix-eupl-compatible-open-source-licences)
- The outputs (like the schema diagram) generated by a (L)GPL licensed component are not (L)GPL licensed. The whole report generated by SchemaSpy does include some SchemaSpy code, and that naturally is covered by the license.
- Lastly, we do not modify (or distribute) SchemaSpy, nor is the application based on SchemaSpy.